docs/reference/remap2/literals/array.cue

```diff
@@ -0,0 +1,22 @@
+package metadata
+
+remap2: literals: array: {
+	title: "Array"
+	description: """
+		An "array" literal is a comma-delimited set of expressions that represent a contiguous growable array type,
+		also known as a 'vector' in Rust.
```

Comment:
I would argue that we should avoid references to Rust whenever possible. I think it's fine in the "preamble" to these docs, and there are places like timestamps where it may be unavoidable (debatable), but I think we should really avoid giving the impression that users need to know *anything* about Rust to use VRL.